.github/workflows/zap-vs-vulnerableapp.yml

```diff
@@ -0,0 +1,17 @@
+name: ZAP vs VulnerableApp
+
+on:
+  workflow_dispatch:
+
+jobs:
+  scan:
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: Start vulnerableApp
+      run: docker run --rm -d -p 9090:8080 --name=owasp-vulnerableapp sasanlabs/owasp-vulnerableapp:latest
```

Comment:
I am not sure how this script will actually run, created this file similar to wavsep.
@psiinon don't we need to first pull the vulnerableApp image ?